regression/systemtest/systemtest_utils.go

```diff
@@ -0,0 +1,97 @@
+package systemtest
+
+import (
+	"os"
+	"io/ioutil"
+	"fmt"
+	"path"
+	"strings"
+	"path/filepath"
+	"gopkg.in/yaml.v2"
+
+	"github.com/hyperledger/fabric-test/tools/operator/testclient/inputStructs"
+	"github.com/hyperledger/fabric-test/tools/operator/networkclient"
+)
+
+//MovePTEReportFile -- Move pteReport file after the test is executed
+func MovePTEReportFile(fileName, logsDir string) {
+	fabricTestDir, _ := GetFabricTestDir()
+	pteScriptsDir := filepath.Join(fabricTestDir, "tools/PTE/CITest/scripts")
+	networkclient.ExecuteCommand("node", []string{filepath.Join(pteScriptsDir, "get_pteReport.js"), "pteReport.txt"}, false)
+	pteReportFilePath := filepath.Join(logsDir, fileName)
+	if _, err := os.Stat(logsDir); os.IsNotExist(err) {
+		os.MkdirAll(logsDir, os.ModePerm)
+	}
+	_ = os.Rename("pteReport.txt", pteReportFilePath)
+}
+
+//GetTestStatusFromReportFile -- To get the status of the test, once it is executed
+func GetTestStatusFromReportFile(filePath, action string) string {
+
+	invokeStatus := "INVOKE Overall TEST RESULTS PASSED"
+	queryStatus := "QUERY Overall TEST RESULTS PASSED"
+	data, err := ioutil.ReadFile(filePath)
+	if err != nil {
+		return fmt.Sprintf("%s", err)
+	}
+	fileData := string(data)
+	switch action {
+	case "invoke":
+		if strings.Contains(fileData, invokeStatus) {
+			return invokeStatus
+		}
+	case "query":
+		if strings.Contains(fileData, queryStatus) {
+			return queryStatus
+		}
+	}
+	return "Overall TEST RESULTS FAILED"
+}
+
+//GetFabricTestDir -- To get the root directory of the fabric-test
+func GetFabricTestDir() (string, error) {
+	currentDir, err := os.Getwd()
+	if err != nil {
+		return currentDir, err
+	}
+	fabricTestPath := path.Join(currentDir, "../../")
+	return fabricTestPath, nil
+}
+
+//ChangeTargetPerrs -- To change the tagetPeers option and creates a temporary test-input.yml file
+func ChangeTargetPerrs(testFilePath, testFileName, targetPeersType string) error {
```

Comment:
is this ChangeTargetPeers instead of `perrs`?

Reply:
Will change this